### PR TITLE
Payments reports

### PR DESCRIPTION
**Pull request by tarunguguloth**
AddedPaymentsModel

### CODE REVIEW

File: model/PaymentModel.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package model
```
Comment (mukesh0513) [inline model/PaymentModel.go]: file name snake_case not camelCase

File: model/PaymentModel.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import (
+"github.com/golang/protobuf/ptypes/timestamp"
+)
+
+type Payments struct {
+	id   string  `json:"id"`
+	Userid  string  `json:"user_id"`
+	Razorpay_user_id string  `json:"razorpay_user_id"`
```
Comment (mukesh0513) [inline model/PaymentModel.go]: gorm tags missing.

Comment (mukesh0513) [reply model/PaymentModel.go]: Struct Variable should be camelCase.

File: model/payment_model.go [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package Model
```
Comment (mukesh0513) [inline model/payment_model.go]: file name does not need to contain model suffix.

**Comment by mukesh0513** [comment]
run gofmt.

**Comment by tarunguguloth** [comment]
Another branch Payment struct was created

**Pull request closed without merging**